### PR TITLE
added myself to contributors

### PR DESCRIPTION
**Pull request by MisterCruz**
*merged all previous branches into one

### CODE REVIEW

File: staticfiles/js/crowdsource.routes.js [hunk 1]
```diff
@@ -140,7 +140,23 @@
     .when('/contributors/arcolife', {
       templateUrl: '/static/templates/contributors/arcolife.html'
     })
-     
+<<<<<<< HEAD
```
Comment (Macbull) [inline staticfiles/js/crowdsource.routes.js]: Can you solve these merge conflicts ? Feel free to ask for any help !

File: staticfiles/templates/contributors/home.html [hunk 2]
```diff
@@ -154,97 +154,99 @@ <h1 class="page-header">
...
-      
+
           <div class="col-sm-2 col-xs-4 thumb">
             <a class="contributor-thumb" href="/contributors/AmitRakesh">
               <img class="thumbnail img-responsive" src="/static/images/contributors/AmitRakesh.jpg" />
-              <span>Amit Rakesh</span> 
+              <span>Amit Rakesh</span>
             </a>
           </div>
+<<<<<<< HEAD
```
Comment (Macbull) [inline staticfiles/templates/contributors/home.html]: Please solve conflicts in this file also !

Comment (MisterCruz) [inline staticfiles/templates/contributors/home.html]: How can I edit it? It won't let me from here, sorry!

Comment (nistala) [inline staticfiles/templates/contributors/home.html]: you would make changes on your branch and push to it, the PR will update automatically

**Comment by MisterCruz** [comment]
@Macbull Fixed!

**Comment by nistala** [comment]
@MisterCruz you still need to merge the branch to develop2

**Comment by MisterCruz** [comment]
making a new PR because of errors

**Pull request closed without merging**